IndyAV/simulation/indyav_gazebo/src/backwheel_plugin.cpp

```diff
@@ -10,29 +10,37 @@ void BackWheelPlugin::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr _sd
 {
   WheelPlugin::Load(_model, _sdf);
 
-  GZ_ASSERT(_sdf->HasElement("max_thrust"), "no max thrust provided");
-  max_thrust_ = _sdf->Get<double>("max_thrust");
+  GZ_ASSERT(_sdf->HasElement("max_torque"), "no max thrust provided");
+  max_torque_ = _sdf->Get<double>("max_torque");
 
   GZ_ASSERT(_sdf->HasElement("max_velocity"), "no max velocity provided");
   max_velocity_ = _sdf->Get<double>("max_velocity");
 
-  base_link_ = model_->GetLink("base_link");
+  back_left_wheel_joint_ = model_->GetJoint("base_link_to_back_left_wheel_link_joint");
```

Comment:
maybe parameterize this by passing in the `"back_left_wheel_link_joint"` and `"back_right_wheel_link_joint"` in the same why that we pass in `max_velocity`?

Reply:
This formatting of this is how it was initially. The compiler freaks out when I say "_sdf->Get<physics::JointPtr>("wheel0");"

Not sure if that function is able to handle that variable type

Reply:
right so are you using _sdf->Get<std::string> to get the name of the link from the sdf?